### PR TITLE
[docs] Mention the new data fetching methods on missing places

### PR DESCRIPTION
**Pull request by lfades**
Read the changes for more details.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| buildDuration | 10.5s | 10.9s | ⚠️ +349ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5272.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.75 kB | 6.75 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| index.html gzip | 917 B | 917 B | ✓ |
| link.html gzip | 926 B | 926 B | ✓ |
| withRouter.html gzip | 915 B | 915 B | ✓ |
| Overall change | 2.76 kB | 2.76 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| buildDuration | 11.1s | 11s | -146ms |
| nodeModulesSize | 47.7 MB | 47.7 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5272.js gzip | 10.2 kB | 10.2 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.3 kB | 56.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.78 kB | 4.78 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.75 kB | 6.75 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _app.js gzip | 1.24 kB | 1.24 kB | ✓ |
| _error.js gzip | 3.15 kB | 3.15 kB | ✓ |
| hooks.js gzip | 664 B | 664 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.03 kB | 2.03 kB | ✓ |
| routerDirect.js gzip | 279 B | 279 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.86 kB | 7.86 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _app.module.js gzip | 594 B | 594 B | ✓ |
| _error.module.js gzip | 2.08 kB | 2.08 kB | ✓ |
| hooks.module.js gzip | 370 B | 370 B | ✓ |
| index.module.js gzip | 212 B | 212 B | ✓ |
| link.module.js gzip | 1.48 kB | 1.48 kB | ✓ |
| routerDirect..dule.js gzip | 271 B | 271 B | ✓ |
| withRouter.m..dule.js gzip | 270 B | 270 B | ✓ |
| Overall change | 5.28 kB | 5.28 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong> Overall decrease ✓</summary>

|  | zeit/next.js canary  | lfades/next.js docs/update-ssr-mentions | Change |
| - | - | - | - |
| _error.js gzip | 233 kB | 232 kB | -454 B |
| 404.html gzip | 1.32 kB | 1.32 kB | ✓ |
| hooks.html gzip | 956 B | 956 B | ✓ |
| index.js gzip | 232 kB | 233 kB | ⚠️ +301 B |
| link.js gzip | 243 kB | 243 kB | -90 B |
| routerDirect.js gzip | 241 kB | 241 kB | -41 B |
| withRouter.js gzip | 241 kB | 241 kB | ✓ |
| Overall change | 1.19 MB | 1.19 MB | -284 B |

</details>

</details>

File: docs/advanced-features/custom-error-page.md [hunk 1]
```diff
@@ -54,25 +54,25 @@ If you want to render the built-in error page you can by importing the `Error` c
 
 ```jsx
 import Error from 'next/error'
-import fetch from 'isomorphic-unfetch'
+import fetch from 'node-fetch'
 
-const Page = ({ errorCode, stars }) => {
+export async function getServerSideProps() {
```
Comment (timneutkens) [inline docs/advanced-features/custom-error-page.md:59]: This is incorrect, _error doesn't exactly support `getServerSideProps` or `getStaticProps` yet as it needs the `err` that is provided in general.

Comment (lfades) [reply docs/advanced-features/custom-error-page.md:59]: @timneutkens This is for the example of a page that wants to import the error page and use it, not for the _error page itself.

Comment (timneutkens) [reply docs/advanced-features/custom-error-page.md:59]: Ah I see! Missed that

**Review by timneutkens — APPROVED** [review]

**Pull request merged**